### ISSUE TITLE
1.5.5 version not compatible with .Net version 4.6.2

### ISSUE THREAD

**Issue by oreporan** [OrleansContrib/OrleansDashboard#100]
```
Error		
Could not install package 'OrleansDashboard 1.5.5'. 
You are trying to install this package into a project that targets '.NETFramework,Version=v4.6.2', 

but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
```

**Comment by richorama** [comment]
Thank you. I'm looking in to it. I hope to have something next week.

**Comment by richorama** [comment]
Fixed in 1.5.6 - sorry about this, the tooling seems to have failed
